concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md

```diff
@@ -0,0 +1,7 @@
+Enable the [repaintChangesOnly]() property to optimize performance in the cases below:
+
+- If you enable editing and use the **columns**.[setCellValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#setCellValue) method.
+
+- If a velocity of real-time updates is high. Refer to the following demo for an example: [Real Time Updates](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RealTimeUpdates).
```

Comment:
What do you mean by `a velocity`?

Reply:
I mean "speed". Should I change it?

Reply:
Or do you mean that 'velocity' is uncountable?

Reply:
```suggestion
- If the speed of real-time updates is high. Refer to the following demo for an example: [Real Time Updates](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RealTimeUpdates).
```